app/src/main/java/com/hapramp/ui/activity/DetailedActivity.java

```diff
@@ -300,7 +300,20 @@ private void renderMarkdown(String body) {
     webView.getSettings().setPluginState(WebSettings.PluginState.ON);
     webView.setWebChromeClient(new WebChromeClient());
     webView.getSettings().setAllowFileAccess(true);
-    webView.setWebViewClient(new WebViewClient());
+    webView.setWebViewClient(new WebViewClient() {
+      @Override
+      public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        if (url != null && (url.matches("(http:\\/\\/www\\.|https:\\/\\/www" +
```

Comment:
Please store the regex in a separate, globally accessible variable so that we can reuse it.